### ISSUE TITLE
Upgrading from 4.1.2 to 4.7.6 causes js error in IE11

### ISSUE THREAD

**Issue by smohadjer** [handlebars-lang/handlebars.js#1690]
Since we upgraded to version 4.7.6 pages which load webcomponents polyfill are throwing error in IE11. I have reported issue on webcomponent's repository, but I thought perhaps some here can also help me figure out what is causing this error. Here is a link to the other ticket:

https://github.com/webcomponents/polyfills/issues/334

The error in IE console says:

```
SCRIPT5007: Object.getOwnPropertyNames: argument is not an Object
webcomponents-bundle.js (77,482)
```

We did not have this error when using handlebars.runtime.js version 4.1.2.

**Comment by nknapp** [comment]
I don't know where this error comes from. Maybe, if you can track down the version that first has the problem. Could you try different versions and tell use which have the problem and which haven't. Then we could look at the changes.

**Comment by al548** [comment]
https://github.com/handlebars-lang/handlebars.js/issues/1690#

Duplicate of #

**Comment by jaylinski** [comment]
Will be fixed when #1776 is tagged and released.

**Comment by jaylinski** [comment]
Closing because handlebars doesn't use `getOwnPropertyNames` anywhere, so this seems to be an issue with `webcomponents/polyfills`.
